sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusClientBuilderTest.java

```diff
@@ -44,6 +46,33 @@
         ENDPOINT, SHARED_ACCESS_KEY_NAME, SHARED_ACCESS_KEY, QUEUE_NAME);
     private static final Proxy PROXY_ADDRESS = new Proxy(Proxy.Type.HTTP, new InetSocketAddress(PROXY_HOST, Integer.parseInt(PROXY_PORT)));
 
+    @Test
+    void viaQueueNameWithTopicNotAllowed() {
+        // Arrange & Act
+        ServiceBusSenderClientBuilder builder = new ServiceBusClientBuilder()
```

Comment:
This is just // Arrange. 
The next part is // Act & Assert.

Same with the section below.